JASP-Desktop/engine/enginerepresentation.cpp

```diff
@@ -60,24 +60,26 @@ void EngineRepresentation::handleEngineCrash()
 
 	switch(_engineState)
 	{
+
 	case engineState::analysis:
 		if(_analysisInProgress)
 		{
-			_analysisInProgress->setErrorInResults("The engine crashed while trying to run this analysis...");
+			QString message = tr("The engine crashed while trying to run this analysis...");
+			_analysisInProgress->setErrorInResults(message.toStdString());
```

Comment:
Why not `_analysisInProgress->setErrorInResults(fq(tr("The engine crashed while trying to run this analysis...")));`?

It isn't really necessary to add the variable here is it...

Reply:
Yes, was ff qutils vergeten